content/admin/Administration Panel/Email.adoc

```diff
@@ -2,15 +2,287 @@
+|*Least Busy*| Select All Users or an existing security group or role. 
+Cases will be assigned to the member of the specified group or role with the least case assignments.
+|*Random*| Select All Users or an existing security group or role. 
+Cases will be assigned randomly to members of the specified group or role.
+|===
+
+*New Case Auto-Reply template* 
+
+If *SuiteCRM* has been configured to auto-create cases, you can select or create an 
+link:../../../user/core-modules/emailtemplates[email template] to use as an automated reponse to 
+notify the sender that a case has been created. If no template is specified here, this automated 
+reponse will not be sent. image:EmailsNewCaseAutoReply.png[New Case Auto-Reply template]
+
+[cols="20,80", frame = "none", grid = "none"]
+|===
+|*No Auto_Reply to this Domain*|No auto-responses will be sent to the specified domain. Use 
+this for example to exclude your company domain, so your users do not receive auto-reply messages. 
+|*Number of Auto-responses*|This setting specifies the maximum number of replies to send to 
+a particular email address in a 24hr period.
+|===
+
+==== Reply To Settings
+
+image:287EmailsGroupReplyToSettings.png[Group Mail Reply To settings]
+
+[cols="20,80", frame = "none", grid = "none"]
+|===
+|*From Address:*| Used as the from address where supported, otherwise the <<Outgoing Mail Configuration, 
+system outbound>> account will be used.
+
+|*Allow users to send emails using the From name and Address as the reply to address:*|When checked, 
+the *From Name* and *From Address* for this account will appear as a *From* option 
+when composing an email for all users that have access to this group account.
+|===
+
+Once configured, all inbound accounts are listed under *Inbound Accounts* on the *Admin* panel, 
+from where they can be edited or removed. 
+
+=== Bounce Handling Account
+
+A Bounce Handling Account is used to manage bounce notifications for an email 
+link:../../../user/core-modules/campaigns[campaign]. Bounced email addresses are recorded 
+in the campaign status.
+
+Once created, the bounce handling account can be selected by users when setting up a 
+campaign.
+
+Select *New Bounce Handling Account* from the Sidebar.
+
+image:EmailsBounceSidebar.png[Sidebar]
+
+Enter the configuration details for the bounce account you are configuring. You will need the 
+username and password for the account, plus the mail server address. 
+Your system administrator will be able to supply these settings.
+
+The mail protocol supported by *SuiteCRM* is IMAP. 
+
+image:EmailsBounceAccountSettings.png[Bounce account settings]
+
+*Monitored Folders* are the folders which are checked for new (unread) mail. *Inbox* and *Trash* 
+folder names must be specified here. 
+
+Click btn:[SELECT] to connect to the mail server and select 
+the relevant folder(s) from the popup dialog.
+
 == Outbound Email
 
```

Comment:
Again, two titles one after the other is confusing, I think that adding some explanatory sentence is better, "to be completed" (or similar)